modules/raft/src/main/java/org/apache/ignite/internal/raft/ExecutorInclinedRaftCommandRunner.java

```diff
@@ -44,6 +46,16 @@ public <R> CompletableFuture<R> run(Command cmd) {
             return future;
         }
 
+        // We can wait for replication completion right here, because client thread waits the entire operation in synchronous API any case.
```

Comment:
```suggestion
        // We can wait for replication completion right here, because client thread waits the entire operation in synchronous API in any case.
```